doc/source/community/contributing.rst

```diff
@@ -222,7 +222,7 @@ This should install all necessary dependencies.
 Once dependencies are in place, make an in-place build by navigating to the git
 clone of the *GeoPandas* repository and running::
 
-    python setup.py develop
+    pip install -e .
```

Comment:
I would either keep the original, or add `--no-build-isolation` to avoid installing the dependencies in a separate build environment.

Reply:
Actually ignore this, that's only relevant if you have to build extensions, for a pure python package like geopandas that shouldn't matter

(I was thinking for a moment this would create a separate build env that needed to install shapely/GEOS and fiona/GDAL, etc)

Reply:
Let's keep the original for now, but it might come up for discussion later. For reference see https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html

Reply:
I would use `pip install -e .` here. At this point of the instructions you already have all the dependencies. I am using `pip` myself for a long time with no issues.